src/app/modules/group/components/member-list/member-list.component.ts

```diff
@@ -42,7 +48,7 @@ export class MemberListComponent implements OnChanges, OnDestroy {
 
   state: 'error' | 'ready' | 'fetching' = 'fetching';
 
-  defaultPolicy: Filter = { type: TypeFilter.Groups };
+  defaultPolicy: Filter = { type: TypeFilter.Groups, directChildren: false };
```

Comment:
There are still several usage of "policy" in this file...

Reply:
fixed